inlong-dashboard/src/ui/pages/Process/Approvals/config.tsx

```diff
@@ -73,6 +74,17 @@ export const getColumns = activedName => [
     dataIndex: 'inlongGroupId',
     render: (text, record) => record.showInList?.inlongGroupId,
   },
+  {
+    title: i18n.t('pages.Approvals.GroupMode'),
+    dataIndex: 'inlongGroupMode',
+    render: (text, record) => {
+      return record.showInList?.inlongGroupMode === 1 ? (
+        <Tag color="green">{i18n.t('pages.Approvals.GroupMode.DataSync')}</Tag>
```

Comment:
I think this can use the system's `StatusTag` component to maintain the unity of the Tag UI of the entire system.

Reply:
Done